lib/macho/load_commands.rb

```diff
@@ -52,8 +52,10 @@ module MachO
     0x2b => :LC_DYLIB_CODE_SIGN_DRS,
     0x2c => :LC_ENCRYPTION_INFO_64,
     0x2d => :LC_LINKER_OPTION,
-    0x2e => :LC_LINKER_OPTIMIZATION_HINT
-  }
+    0x2e => :LC_LINKER_OPTIMIZATION_HINT,
+    0x2f => :LC_VERSION_MIN_TVOS,
+    0x30 => :LC_VERSION_MIN_WATCHOS
```

Comment:
Yeah, I have a bad habit of not doing that. I'll re-add them as I continue to change the relevant parts of these files.